terraform/modules/cluster/mig-with-container/main.tf

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+locals {
+  region = join("-", slice(split("-", var.zone), 0, 2))
+
+  machine_image = {
+    project = "cos-cloud"
+    family  = "cos-stable"
```

Comment:
people can choose to use different COS images. So we should not restrict this. We should only restrict on the project to be `cos-cloud`